docs/repo/architecture.md

```diff
@@ -1,6 +1,8 @@
 # Architecture
 
-![image](https://cloud.githubusercontent.com/assets/1103906/14921086/248166fa-0de5-11e6-86fa-e96240b7fd07.png)
+|||
+|:-------|:----------|
+| ![image](https://cloud.githubusercontent.com/assets/9797472/26602516/9481c572-4538-11e7-971d-cbb9935a40d7.png) | ![image](https://cloud.githubusercontent.com/assets/9797472/26602535/a9e7c560-4538-11e7-823f-b3471b2c26fb.png)|
```

Comment:
The first image is missing the `.VS` namespace in the `MS.VS.ProjectSystem.CSharp.VS`. This is the case for all the instances.